index.md

```diff
@@ -1,19 +1,8 @@
 ---
 layout: HubPage
 hide_bc: true
-
 title: .NET Documentation
-description:
-keywords:
-author:
-manager:
-ms.date: 11/14/2016
-ms.topic: hubpage
-ms.prod:
-ms.service:
-ms.technology:
-ms.suite:
-ms.assetid:
+description: Learn about .NET
```

Comment:
we should probably add a more meaningful description if we're going to add this metadata.